### PR TITLE
remove {{event}} macros for Event_handlers

### PR DESCRIPTION
**Pull request by mfuji09**
> What was wrong/why is this fix needed? (quick summary only)

replace {{event}} macros for Event_handlers with normal links because they aren't keywords in codes.

> Issue number (if there is an associated issue)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/mdn/structures/macros/commonly-used_macros/index.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h3 id="Linking_to_pages_in_references">Linking to pages in references</h3>
       <tr>
         <td><a href="https://github.com/mdn/yari/blob/main/kumascript/macros/event.ejs">Event</a>.</td>
         <td><a href="/en-US/docs/Web/Events">Events reference</a> (/Web/Events)</td>
-        <td><code>\{{Event("Event_handlers", "Event handlders")}}</code> results in {{Event("Event_handlers", "Event handlers")}}
+        <td><code>\{{Event("Event_handlers", "Event handlers")}}</code> results in {{Event("Event_handlers", "Event handlers")}}
```
Comment (teoli2003) [inline files/en-us/mdn/structures/macros/commonly-used_macros/index.html:121]: I don't think it is useful to have this example, as we don't want to link to _event handlers_ that way.

I'm tempted to remove the whole `<tr>` here.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Beside the entry in the meta documentation, all the other changes looks good.

**Comment by teoli2003** [comment]
And for the record, this will partially deal with #5555.

File: files/en-us/mdn/structures/macros/commonly-used_macros/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/mdn/structures/macros/commonly-used_macros/index.html:121]:
```suggestion
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/mdn/structures/macros/commonly-used_macros/index.html [hunk 2]
```diff
@@ -118,7 +118,6 @@ <h3 id="Linking_to_pages_in_references">Linking to pages in references</h3>
       <tr>
         <td><a href="https://github.com/mdn/yari/blob/main/kumascript/macros/event.ejs">Event</a>.</td>
         <td><a href="/en-US/docs/Web/Events">Events reference</a> (/Web/Events)</td>
-        <td><code>\{{Event("Event_handlers", "Event handlers")}}</code> results in {{Event("Event_handlers", "Event handlers")}}
```
Comment (teoli2003) [inline files/en-us/mdn/structures/macros/commonly-used_macros/index.html]: I think this is even better as it keeps the warning.

You need to keep the `<td>`, though.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/mdn/structures/macros/commonly-used_macros/index.html [hunk 3]
```diff
@@ -118,7 +118,6 @@ <h3 id="Linking_to_pages_in_references">Linking to pages in references</h3>
       <tr>
         <td><a href="https://github.com/mdn/yari/blob/main/kumascript/macros/event.ejs">Event</a>.</td>
         <td><a href="/en-US/docs/Web/Events">Events reference</a> (/Web/Events)</td>
-        <td><code>\{{Event("Event_handlers", "Event handlders")}}</code> results in {{Event("Event_handlers", "Event handlers")}}
         <div class="notecard note">
```
Comment (teoli2003) [inline files/en-us/mdn/structures/macros/commonly-used_macros/index.html:121]:
```suggestion
        <td>
        <div class="notecard note">
```

Comment (mfuji09) [reply files/en-us/mdn/structures/macros/commonly-used_macros/index.html:122]: I see, thank you!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Nice!

**Comment by hamishwillee** [comment]
Awesome! I've added a note to #5555 that perhaps we should remove the macro now from yari

**Pull request merged**